std/stdio.d

```diff
@@ -1623,7 +1620,7 @@ is recommended if you want to process a complete file.
     {
         import std.algorithm.mutation : swap;
         import std.algorithm.searching : endsWith;
-        import std.range.primitives : back, empty;
+        import std.range.primitives : back;
```

Comment:
Left this one range function here because it's the only use of `back` in the entire module.